polynote-frontend/style/styles.less

```diff
@@ -2,28 +2,28 @@
   font-family: Hasklig;
   font-weight: 400;
   font-style: normal;
-  src: url(fonts/Hasklig-Regular.otf) format("otf");
+  src: url(/style/fonts/Hasklig-Regular.otf) format("opentype");
```

Comment:
I think it was the `otf` vs `opentype` thing that was causing Chrome to refuse to load it. Not the relative URL. But anyway...